src/Mailgun/Resource/Api/Suppressions/Bounce/ShowResponse.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Resource\Api\Suppressions\Bounce;
+
+use Mailgun\Resource\ApiResponse;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class ShowResponse implements ApiResponse
+{
+    /**
+     * @var string
+     */
+    private $address;
+
+    /**
+     * @var string
+     */
+    private $code;
+
+    /**
+     * @var string
+     */
+    private $error;
+
+    /**
+     * @var \DateTime
+     */
+    private $created_at;
+
+    /**
+     * @param string $address
+     * @param string $code
+     * @param string $error
+     */
+    public function __construct($address)
+    {
+        $this->address = $address;
+        $this->created_at = new \DateTime();
+    }
+
+    /**
+     * @param array $data
+     *
+     * @return Bounce
```

Comment:
`$bounce` variable is an instance of `ShowResponse` due you're calling `new self()`. Can you update the annotation please? Thanks,